docs/artifactory/tokencreate/help.go

```diff
@@ -0,0 +1,8 @@
+package tokencreate
+
+const Description = "Creates an access token. By default an user-scoped token will be created, unless groups and/or admin-privileges are specified."
```

Comment:
...unless groups and/or admin-privileges are specified. --> ...unless the --groups and/or --grant-admin options are specified.